docs/release_notes/2024/August_2024/new-ui-sql.md

```diff
@@ -43,7 +43,7 @@ We've added the powerful functionality to configure variables directly in a [SQL
 
 ### Building SCD2 Tables
 
-Slowly changing dimensions (SCD) doesn't have to be so challenging. With this release, you can take advantage of Prophecy's visual interface to configure [SCD type 2 writes](../../../SQL/development/target-models/write-options#scd-2.md) and retain data history. Open the Target Model Gem, select Table format. Just select a unique key and the relevant timestamp column for your data. Now any data practitioner can capture updated records without deleting the previous records.
+Slowly changing dimensions (SCD) doesn't have to be so challenging. With this release, you can take advantage of Prophecy's visual interface to configure [SCD type 2 writes](../../../SQL/development/target-models/write-options#scd-2) and retain data history. Open the Target Model Gem, select Table format. Just select a unique key and the relevant timestamp column for your data. Now any data practitioner can capture updated records without deleting the previous records.
```

Comment:
Suggest: 
select Table format. For the merge write option, just select SCD2 and chose a unique key and relevant timestamp column for your data. ...

Reply:
Done.